{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "total_count": 2,
    "last_updated": "2025-08-05T05:45:39.222032",
    "severity_distribution": {
      "critical": 1,
      "high": 0,
      "medium": 1,
      "low": 0,
      "none": 0
    }
  },
  "cves": [
    {
      "id": "CVE-2025-54982",
      "publishedDate": "2025-08-05T05:36:06.752Z",
      "lastModifiedDate": "2025-08-05T05:36:06.752Z",
      "description": "An improper verification of cryptographic signature in Zscaler's SAML authentication mechanism on the server-side allowed an authentication abuse.",
      "severity": 9.6,
      "references": [
        {
          "url": "https://help.zscaler.com/zia/about-identity-providers",
          "type": "reference"
        }
      ],
      "affected": [
        {
          "defaultStatus": "unaffected",
          "product": "Authentication Server",
          "vendor": "Zscaler",
          "versions": [
            {
              "lessThan": "6.2r",
              "status": "affected",
              "version": "0",
              "versionType": "custom"
            }
          ]
        }
      ],
      "problemType": [
        "CWE-347 Improper Verification of Cryptographic Signature"
      ],
      "fix_suggestion": "无法生成修复建议"
    },
    {
      "id": "CVE-2025-8547",
      "publishedDate": "2025-08-05T05:32:06.564Z",
      "lastModifiedDate": "2025-08-05T05:32:06.564Z",
      "description": "A vulnerability has been found in atjiu pybbs up to 6.0.0 and classified as critical. This vulnerability affects unknown code of the component Email Verification Handler. The manipulation leads to improper authorization. The attack can be initiated remotely. The exploit has been disclosed to the public and may be used. The name of the patch is 044f22893bee254dc2bb0d30f614913fab3c22c2. It is recommended to apply a patch to fix this issue.",
      "severity": 5.3,
      "references": [
        {
          "url": "https://vuldb.com/?id.318676",
          "type": "reference"
        },
        {
          "url": "https://vuldb.com/?ctiid.318676",
          "type": "reference"
        },
        {
          "url": "https://vuldb.com/?submit.622180",
          "type": "reference"
        },
        {
          "url": "https://github.com/atjiu/pybbs/issues/200",
          "type": "reference"
        },
        {
          "url": "https://github.com/atjiu/pybbs/issues/200#issuecomment-3134710486",
          "type": "reference"
        },
        {
          "url": "https://github.com/atjiu/pybbs/issues/200#issue-3256283647",
          "type": "poc"
        },
        {
          "url": "https://github.com/atjiu/pybbs/commit/044f22893bee254dc2bb0d30f614913fab3c22c2",
          "type": "reference"
        }
      ],
      "affected": [
        {
          "vendor": "atjiu",
          "product": "pybbs",
          "versions": [
            {
              "version": "6.0",
              "status": "affected"
            }
          ],
          "modules": [
            "Email Verification Handler"
          ]
        }
      ],
      "problemType": [
        "Improper Authorization",
        "Incorrect Privilege Assignment"
      ],
      "fix_suggestion": "无法生成修复建议"
    }
  ]
}